ITSeed-Official/itseed_backend
# [init] Use strapi to set up the backend project

## Summary
Since we want to build the APIs and CMS using `strapi`, I followed the  [official document](https://strapi.io/documentation/developer-docs/latest/setup-deployment-guides/installation/docker.html#creating-a-strapi-project) and used docker to create a new strapi project.

## How to test it
1. git clone this project
2. docker-compose pull
3. docker-compose up -d
4. go to [http://localhost:1337/](http://localhost:1337/), and you can see this page.
![image](https://user-images.githubusercontent.com/33183531/132988429-e3d4a89f-0a2d-49bc-bfcb-5ac6a0f10999.png)
